packages/shared-react/src/api/queries/addressQueries.ts

```diff
@@ -105,7 +105,7 @@ export const addressTokensBalancesQuery = ({ addressHash, networkId, skip }: Add
 
 export const addressBalancesQuery = ({ addressHash, networkId, skip }: AddressLatestTransactionQueryProps) =>
   queryOptions({
-    queryKey: ['address', addressHash, 'balance', { networkId }],
+    queryKey: ['address', addressHash, 'level:1', 'balances-all', { networkId }],
```

Comment:
I may need some explanation regarding those keys.

Two questions popping in my mind:
- How are those "levels" defined?
- Why are we sometimes using camel case (`searchString`, `tokensByType`), and other times dashes (`balances-all`, `tokens-search-strings`)?

Reply:
level:1 needs to be invalidated before level:2, because the queryFn of level:2 is calling queries that have level:1.

I did not find a better way to solve this. Using numbers creates some conceptual idea of dependency.

As for the camel case, I just didn't notice it. I'll fix that.